src/it/scala/com/exasol/cloudetl/DockerNamedNetwork.scala

```diff
@@ -0,0 +1,81 @@
+package com.exasol.cloudetl
+
+import java.util.concurrent.ConcurrentHashMap
+
+import org.apache.spark.internal.Logging
+
+import org.junit.runner.Description
+import org.junit.runners.model.Statement
+import org.testcontainers.DockerClientFactory
+import org.testcontainers.containers.Network
+import org.testcontainers.utility.ResourceReaper
+
+/**
+ * A reusable docker network.
+ *
+ * At the moment, the docker container {@code reuse} is ignored when a
+ * network is attached to a container. This class creates docker network
+ * that can be attached to reusable container.
+ *
+ * @param name name of the docker network
+ * @param reuse boolean value to indicate reusability
+ */
+class DockerNamedNetwork(name: String, reuse: Boolean) extends Network with Logging {
```

Comment:
I think you already had this in a different project?
If so I think you should avoid duplicating it.
Maybe you could even move it into exasol-testcontainer or at least into a library.

Reply:
Yes, it is exact copy.

I totally agree. I have created a ticket for it: https://github.com/exasol/exasol-testcontainers/issues/128